pandas/tests/io/json/test_pandas.py

```diff
@@ -991,26 +864,28 @@ def test_convert_dates(self):
         result = read_json(json, typ="series")
         assert_series_equal(result, ts)
 
-    def test_convert_dates_infer(self):
-        # GH10747
-        from pandas.io.json import dumps
-
-        infer_words = [
+    @pytest.mark.parametrize(
```

Comment:
will give another go at reviewing the check_all_orients stuff.  If this stalls and you _do_ decide to break this up, everything below this point LGTM.